napari/layers/points/points.py

```diff
@@ -1260,6 +1260,26 @@ def _get_value(self, position) -> Union[None, int]:
 
         return selection
 
+    def _get_value_3d(self, **kwargs):
+        """Get the layer data value along a ray. Currently, this just
+        returns None. We will implement this in future work.
```

Comment:
```suggestion
        """Get the layer data value along a ray.
        
        Currently, this just returns None. We will implement this
        in future work.
```

[PEP-257](https://www.python.org/dev/peps/pep-0257/): "Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line."